storage/CSBTreeIndexSubBlock.cpp

```diff
@@ -353,6 +355,12 @@ std::size_t CSBTreeIndexSubBlock::EstimateBytesPerTuple(
   return (5 * key_length) >> 1;
 }
 
+std::size_t CSBTreeIndexSubBlock::EstimateBytesPerBlock(
+    const CatalogRelationSchema &relation,
+    const IndexSubBlockDescription &description) {
+  return 0;
```

Comment:
Maybe we should make a constant in `IndexSubBlock.hpp` to describe when an `EstimateSize` function is not applicable...

`size_t kSizeNotApplicable = 0` ?